flepimop/gempyor_pkg/src/gempyor/config_validator.py

```diff
@@ -70,19 +70,31 @@ def validate_seedingfile(cls, values):
         seeding_file_type = values.get('seeding_file_type')        
         seeding_file = values.get('seeding_file')        
         if method == 'PoissonDistributed' and not lambda_file:
-            raise ValueError(f'Error in Seeding: A lambda_file is required when method is {method}')
+            raise ValueError(
+                f"A `lambda_file` is required when method is '{method}', "
+                f"was given '{lambda_file}'."
+            )
         if method == 'FolderDraw' and not seeding_file_type:
-            raise ValueError('Error in Seeding: A seeding_file_type is required when method is FolderDraw')
+            raise ValueError(
+                f"A `seeding_file_type` is required when method is 'FolderDraw', "
+                f"was given '{seeding_file_type}'."
+            )
         if method == 'FromFile' and not seeding_file:
-            raise ValueError('Error in Seeding: A seeding_file is required when method is FromFile')
+            raise ValueError(
+                f"A `seeding_file` is required when method is 'FromFile', "
+                f"was given '{seeding_file}'."
+            )
```

Comment:
bit odd to use {method} in first error, but not subsequent items.

Reply:
also ... aren't all the "was given" items different flavors of falsey? seems like that part of the error can be dropped.